composer.json

```diff
@@ -16,13 +16,13 @@
         }
     ],
     "require": {
-	"php": "^7.1",
-        "symfony/framework-bundle": "^4.0 || ^5.0"
+        "php": "^7.3|^8.0",
```

Comment:
I'd suggest using ">=7.3" constraint instead, if PHP9 were to be released soon (not going to happen, BUT) this bundle would probably be compatible out of the box.

Reply:
I wouldn't recommend this. It would be easy to submit a new PR for this change. And if a new PHP Version is released, it should be covered by CI.